src/api/routes/babamul/users.rs

```diff
@@ -500,7 +576,7 @@ pub struct BabamulAuthResponse {
 #[utoipa::path(
     post,
     path = "/babamul/auth",
```

Comment:
I probably would have the module path mimic the URL path and not put this in `users.rs`; maybe just in `mod.rs` since it's a top level Babamul endpoint.